src/Node/Express/Middleware/CookieParser.purs

```diff
@@ -9,6 +9,12 @@ import Node.Express.Types (Request, Response, ExpressM)
 
 foreign import _cookieParser :: forall eff. Fn3 Request Response (ExpressM eff Unit) (ExpressM eff Unit)
 
+foreign import _secretCookieParser :: forall eff. String -> Fn3 Request Response (ExpressM eff Unit) (ExpressM eff Unit)
+
 cookieParser :: forall eff. Handler eff
 cookieParser = HandlerM $
   \req res nxt -> liftEff $ runFn3 _cookieParser req res nxt
+
+secretCookieParser :: forall eff. String -> Handler eff
```

Comment:
Could you add here a bit of documentation describing parameters and packages necessary to be installed in order to use this middleware?